### PR TITLE
Add Sophie's short bio

### LINKED ISSUE #219: Add profile for Sophie
Sophie to add profile for about us. 
Ensure image is correct dimensions.

### PR DESCRIPTION
**Pull request by sophie22**
resolves #219

### CODE REVIEW

File: _team_member/sophie.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+team-member-photo: /assets/img/team/SophieR.jpeg
+job-role: Clinical Scientist in AI
+---
+
+### Bio
+Sophie joined the CSC team in June 2023. Previously she worked as a Clinical Scientist in Genomics Bioinformatics at Addenbrooke's Hospital in Cambridge, having finished the Scientist Training Programme in 2022. She has worked on numerous service improvement and clinical software development projects within the Clinical Genetics department, validating and deploying new variant detection software and contributing to routine bioinformatic analysis of patient DNA sequencing data for predictive and diagnostic genomic testing.
+
+
+### CSC Team Role
+Sophie is currently working on improvements to Hazen, a quality assurance framework for MR imaging, and its web app version. She is also being trained to build, train, evaluate and deploy machine learning models. As her first project, Sophie is working on the MRI wrist fracture detection model with Laurence and Dika.
```
Comment (AnilMistry) [inline _team_member/sophie.md]: I would change this to: 

Sophie is currently working on improvements to Hazen, a quality assurance framework for MR imaging, and its web app version. She is also involved in building, training, evaluating and deploying machine learning models, leading on the MRI wrist fracture detection model.

**Review by AnilMistry — CHANGES_REQUESTED** [review]

**Comment by sophie22** [comment]
updated role description as requested

**Comment by AnilMistry** [comment]
Apoliges, i have missed something here, could you also fill in -data/team.yml file, filling out a new section for yourself?

File: _data/team.yml [hunk 2]
```diff
@@ -101,30 +101,36 @@ en: &DEFAULT_EN
...
         role: "Data Scientist"
         team-member-page: /team_member/agathe.html
         image: assets/img/team/Agathe.jpg
         social:
           - url: https://www.linkedin.com/in/agathe-zecevic-7034b91b0/
             icon: fab fa-linkedin-in
+      - name: "Sophie Ratkai"
+        role: "Clinical Scientist"
+        team-member-page: /team_member/sophie.html
+        image: assets/img/team/SophieR.png
```
Comment (AnilMistry) [inline _data/team.yml]: This should be image: assets/img/team/SophieR.jpeg

**Review by AnilMistry — CHANGES_REQUESTED** [review]
final issue to resolve, then we're good. After committing the change can you re-request me as a review on the PR?

**Review by AnilMistry — APPROVED** [review]
brilliant, works locally for me.

**Pull request merged**